tests/func/test_api.py

```diff
@@ -128,7 +128,7 @@ def test_open_granular(tmp_dir, dvc, remote):
     [
         pytest.lazy_fixture(cloud)
         for cloud in [
-            "real_s3",  # NOTE: moto's s3 fails in some tests
+            "s3",
```

Comment:
Not sure why it fails on the CI as it is passing locally. From the error, it seems that the response is empty.